### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by vjeffrey**
This commit adds more testing!

A few of them I couldn't get to work correctly....

### CODE REVIEW

File: files/default/audit_report.rb [hunk 1]
```diff
@@ -15,15 +15,16 @@ def report
         interval_enabled = node['audit']['interval']['enabled']
         interval_time = node['audit']['interval']['time']
         report_file = node['audit']['output']
+        test_array = node['audit']['profiles']
```
Comment (chris-rock) [inline files/default/audit_report.rb]: Is `profiles` maybe a better name? I was thinking about an array of single tests first

File: spec/unit/libraries/default_spec.rb [hunk 2]
```diff
@@ -20,6 +20,9 @@
 require 'spec_helper'
```
Comment (chris-rock) [inline spec/unit/libraries/default_spec.rb]: We may should rename the spec to `visibility_spec.rb`. Do we intent to add more specs for other reporters?

File: files/default/audit_report.rb (hunk 1 above)
Comment (vjeffrey) [inline files/default/audit_report.rb]: yup! :+1:

File: spec/unit/libraries/compliance_spec.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# encoding: utf-8
+#
+# Cookbook Name:: audit
+# Spec:: compliance
+
+require 'spec_helper'
+require_relative '../../../libraries/collector_classes'
+
+# TODO: can't get this to work: `# ./libraries/collector_classes.rb:251:in `node_info'`
+# need to figure out how to mock the run context
```
Comment (vjeffrey) [inline spec/unit/libraries/compliance_spec.rb]: :( can't figure out how to mock the run context..is there another way around this?

Comment (chris-rock) [inline spec/unit/libraries/compliance_spec.rb]: The solution is the removal of the run context. I'll work on a PR to cover that

Comment (vjeffrey) [inline spec/unit/libraries/compliance_spec.rb]: thank you!!

**Comment by chris-rock** [comment]
This fixes https://github.com/chef-cookbooks/audit/issues/128

**Pull request merged**